File: src/dataset/cpp/raindrops_exercism-felbit/claude/ClaudeSonnet4_raindrops_v4.cpp
#include "raindrops.h"

namespace raindrops
{
    std::string convert(int number)
    {
        std::string result;
        result.reserve(15);

        const bool div3 = (number % 3 == 0);
        const bool div5 = (number % 5 == 0);
        const bool div7 = (number % 7 == 0);

        if (div3) result += "Pling";
        if (div5) result += "Plang";
        if (div7) result += "Plong";

        if (!(div3 || div5 || div7)) {
            result = std::to_string(number);
        }

        return result;
    }
}

File: src/dataset/typescript/raindrops_exercism-wackerow/claude/ClaudeSonnet4_raindrops_v4.ts
export default class Raindrops {
  public convert(num: number): string {
    const div3 = num % 3 === 0;
    const div5 = num % 5 === 0;
    const div7 = num % 7 === 0;
    
    if (!div3 && !div5 && !div7) {
      return num.toString();
    }
    
    return (div3 ? 'Pling' : '') + (div5 ? 'Plang' : '') + (div7 ? 'Plong' : '');
  }
}